src/MudBlazor/Components/Avatar/MudAvatar.razor.cs

```diff
@@ -51,14 +51,14 @@ partial class MudAvatar : MudComponentBase, IDisposable
         /// Link to image, if set a image will be displayed instead of text.
         /// </summary>
         [Parameter]
-        [Category(CategoryTypes.Avatar.Behavior)]
+        [Obsolete("Add a MudImage as the ChildContent instead", true)]
```

Comment:
```suggestion
        [Obsolete("Add a MudImage as the ChildContent instead", false)]
```
Let's only show a warning until we remove the support completely